rebasehelper/plugins/spec_hooks/replace_old_version.py

```diff
@@ -114,18 +99,33 @@ def run(cls, spec_file: SpecFile, rebase_spec_file: SpecFile, **kwargs: Any):
         replace_with_macro = bool(kwargs.get('replace_old_version_with_macro'))
 
         subversion_patterns = cls._create_possible_replacements(old_version, new_version, replace_with_macro)
+        examined_lines = collections.defaultdict(set)
+        for tag in rebase_spec_file.tags.filter():
```

Comment:
This still iterates only through preamble. Is that intentional?

Reply:
Oh, it defaults to '%package'. Is there an easy way how to iterate over all tags or do I have to use `Tags.items` attribute to do so?